pkg/cmdutil/cmdutil.go

```diff
@@ -38,3 +44,54 @@ func StringSliceToListStringWithQuotes(validOptions []string) string {
 	}
 	return listF
 }
+
+// IsURL accepts a string and determines if it is a URL
+func IsURL(s string) bool {
+	return strings.HasPrefix(s, "http:/") || strings.HasPrefix(s, "https:/")
+}
+
+// GetContentFromFileURL loads file content from the provided URL
+func GetContentFromFileURL(url string, ctx context.Context) (*os.File, error) {
```

Comment:
FYI We already have some helper that has similar content: CreateFileFromStdin.

Reply:
https://github.com/redhat-developer/app-services-cli/blob/analitics/pkg/cmd/registry/artifact/util/files.go#L12:L13